build/get-own-property-symbols.max.js

```diff
@@ -20,15 +20,15 @@ OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 THE SOFTWARE.
 
 */
-(function (Object, GOPS) {'use strict';
+function defineGetOwnPropertySymbols(Object, GOPS) {'use strict';
```

Comment:
This file is build output; it should never be updated as part of a PR. Please revert it.